src/overlays/actors/ovl_En_Knight/z_en_knight.h

```diff
@@ -1,29 +1,155 @@
+    /* 0x15E */ s16 effectTimer;
+    /* 0x160 */ s16 breathAlpha;
+    /* 0x162 */ u8 isHeadless;
+    /* 0x164 */ Vec3f afterImageOffset;
+    /* 0x170 */ s16 pitchToPlayer;
+    /* 0x172 */ s16 yawToPlayer;
+    /* 0x174 */ s16 leftLegUpperRotation;
+    /* 0x176 */ s16 rightLegUpperRotation;
+    /* 0x178 */ s16 leftLegLowerRotation;
+    /* 0x17A */ s16 rightLegLowerRotation;
+    /* 0x17C */ f32 bodyAlpha;
+    /* 0x180 */ f32 shadowAlphaFactor;
+    /* 0x184 */ s16 neckPitch;
+    /* 0x186 */ s16 neckPitchTarget;
+    /* 0x188 */ s16 neckRoll;
+    /* 0x18A */ s16 neckRollTarget;
+    /* 0x18C */ s16 neckRotationMaxStep;
+    /* 0x18E */ s16 neckRotationStepScale;
+    /* 0x190 */ u8 swordColliderActive;
+    /* 0x191 */ u8 shieldColliderActive;
+    /* 0x192 */ u8 canRetreat;
+    /* 0x193 */ u8 shieldingInvulnerabilityTimer;
+    /* 0x194 */ SkelAnime skelAnime;
+    /* 0x1D8 */ f32 animLastFrame;
+    /* 0x1DC */ Vec3f bodyPartsPos[15];
+    /* 0x290 */ u8 igosCurtainReaction;
+    /* 0x291 */ u8 lightRayDamageTimer;
+    /* 0x292 */ u8 retreatWhileShielding;
+    /* 0x294 */ f32 dmgEffectScale;
+    /* 0x298 */ f32 dmgEffectIceMeltingScale;
+    /* 0x29C */ f32 dmgEffectAlpha;
+    /* 0x2A0 */ u8 dmgEffectType;
+    /* 0x2A1 */ u8 dmgEffectState;
+    /* 0x2A2 */ s16 dmgEffectDuration;
+    /* 0x2A4 */ f32 animTranslationX;
+    /* 0x2A8 */ f32 animTranslationZ;
+    /* 0x2AC */ Vec3f feetPositions[2];
+    /* 0x2C4 */ Vec3s jointTable[KNIGHT_LIMB_MAX];
+    /* 0x372 */ Vec3s morphTable[KNIGHT_LIMB_MAX];
     /* 0x420 */ EnKnightActionFunc actionFunc;
-    /* 0x424 */ char unk_424[0x25C];
-    /* 0x680 */ u8 unk_680;
-    /* 0x681 */ char unk_681[0x9];
-    /* 0x68A */ s16 unk_68A;
-    /* 0x68C */ char pad68C[0x48];
+    /* 0x424 */ EnKnightActionFunc prevActionFunc;
+    /* 0x428 */ s16 jawRotation;
+    /* 0x42C */ f32 jawRotationAmplitude;
+    /* 0x430 */ f32 jawRotationAmplitudeTarget;
+    /* 0x434 */ Vec3f breathBasePos;
+    /* 0x440 */ Vec3f retreatTowards;
+    /* 0x44C */ u8 blureState;
+    /* 0x450 */ f32 blureAlpha;
+    /* 0x454 */ Vec3f blureTranslation;
+    /* 0x460 */ Vec3f blureRotation;
+    /* 0x46C */ f32 swordScale;
+    /* 0x470 */ f32 shieldScale;
+    /* 0x474 */ f32 shieldLightReflectionScale;
+    /* 0x478 */ s16 shieldHitTimer;
+    /* 0x47C */ Vec3f shieldParticlesPos;
+    /* 0x488 */ ColliderCylinder headAttackCollider;
+    /* 0x4D4 */ ColliderJntSph swordCollider;
+    /* 0x4F4 */ ColliderJntSphElement swordColliderElements[1];
+    /* 0x534 */ ColliderJntSph shieldCollider;
+    /* 0x554 */ ColliderJntSphElement shieldColliderElements[1];
+    /* 0x594 */ ColliderJntSph bodyCollider;
+    /* 0x5B4 */ ColliderJntSphElement bodyColliderElements[2];
+    /* 0x634 */ ColliderCylinder headCollider;
+    /* 0x680 */ u8 inCurtainReaction;
+    /* 0x684 */ u32 csTimer;
+    /* 0x688 */ s16 csState;
+    /* 0x68A */ s16 csCamId;
+    /* 0x68C */ Vec3f csCamEye;
+    /* 0x698 */ Vec3f csCamAt;
```

Comment:
```suggestion
    /* 0x68A */ s16 subCamId;
    /* 0x68C */ Vec3f subCamEye;
    /* 0x698 */ Vec3f subCamAt;
```

Reply:
I don't agree with `csCam` -> `subCam`. `csCam` is more precise, it's a sub-camera that is specifically used for cutscenes

Reply:
While that is true, it does seem that every other actor currently just uses `subCam` so I would agree with darkeye on this one, at least for consistency with the rest of the codebase. (Also looking on discord looks like this discussion has been had before https://discord.com/channels/688807550715560050/688851317593997489/974277614917283891)

Reply:
I've made the change but I still disagree with it. I think this should be reconsidered more widely.